### ISSUE TITLE
ng-click's not working in AngularJS after upgrading from OnsenUI 2.0.4

### ISSUE THREAD

**Issue by gogodigimo** [OnsenUI/OnsenUI#2170]
__Environment__ 

```
[Core]
    onsenui 2.0.5 (and beyond)

[Framework]
    angular1 1.5.9

[Framework binding]
    angular-onsenui 2.0.5 (and beyond)

[Additional libraries]
    none
    BounceJS (http://bouncejs.com)
    FlipClock (http://flipclockjs.com)
    Pusher (http://pusher.com)
    Swiper (http://www.idangero.us/swiper)

[Platform]
    Mobile - Android 7.1

[Browser]
    Mobile - Android WebView
```

__Encountered problem__
When I upgraded from OnsenUI 2.0.4 to 2.0.5 or anything else beyond that, all of my ng-click events fail on listitems, buttons, images and more.

__How to reproduce__
With the simple code 
  <ons-list-item modifier="chevron" ng-click="GoTo_StartNewGame()">
    Start Game
  </ons-list-item>
my GoTo_StartNewGame function within the AngularJS scope that is called in OnsenUI 2.0.4 is not called in the next dist release 2.0.5, nor any further releases.

**Comment by frandiox** [comment]
@gogodigimo Thanks for reporting. Does [this code work](https://codepen.io/frankdiox/pen/bryKmO?editors=1010) for you? It works at least in desktop browsers.

**Comment by gogodigimo** [comment]
Not when I put it in my current app, but it's possible there's a conflict with one of the additional libraries I'm using. I'll start a new project and try it out.

**Comment by gogodigimo** [comment]
So, I think I've figured out the source of the problem. I was loading the app using 
`var module = angular.module('my-app', ['onsen']);` 
with the ng-app in my html attributes:
`<html lang="en" ng-app="my-app">`

When I remove those and change my controller declaration from:
`module.controller('AppController', function ($scope) {`
to
`ons.bootstrap().controller('AppController', function ($scope) {`
it works just fine.

Forgive my ignorance, but is there a way now to import other angular libraries into the module?
In other words, how would I emulate the following using ons.bootstrap()?
`var module = angular.module('app', ['onsen.directives', 'ngSanitize', 'ngAnimate', 'ngAria', 'ngMessages', 'ngTouch', 'ngMaterial']);`

**Comment by frandiox** [comment]
Yep, reference [here](https://onsen.io/v2/api/angular1/ons.html#method-bootstrap). 👍
